src/component/page/deltakerliste-detaljer/DeltakerlisteDetaljerPage.module.scss

```diff
@@ -27,6 +27,7 @@
 }
 
 .infoSection {
+	width: 20rem;
```

Comment:
Denne er for at siden ikke skal hoppe sånn når vi legger til filter/chips. Burde sjekkes da min skjerm har lav oppløsning og ikke så mye plass, mens deres har nok høy oppløsning og liten tekst